lavalink/NodeManager.py

```diff
@@ -6,9 +6,9 @@
 from .WebSocket import WebSocket
 
 log = logging.getLogger(__name__)
-DISCORD_REGIONS = ["amsterdam", "brazil", "eu-central", "eu-west", "frankfurt", "hongkong", "japan", "london", "russia",
-                   "singapore", "southafrica", "sydney", "us-central", "us-east", "us-south", "us-west",
-                   "vip-amsterdam", "vip-us-east", "vip-us-west"]
+DISCORD_REGIONS = ('amsterdam', 'brazil', 'eu-central', 'eu-west', 'frankfurt', 'hongkong', 'japan', 'london', 'russia',
```

Comment:
Not sure if this would fare better being a dict?

Reply:
It's static, just for the check if region exists if the user is setting them manually.

Reply:
Ah okay